settings/dev.py

```diff
@@ -10,12 +10,7 @@
     'debug_toolbar.middleware.DebugToolbarMiddleware',
 ]
 
-ALLOWED_HOSTS = [
-    'api-staging-art.andela.com',
-    '0.0.0.0',
-    '127.0.0.1',
-    'art-backend.herokuapp.com'
-]
+ALLOWED_HOSTS = [os.environ.get('MY_HOST_IP'), 'api-staging-art.andela.com']
```

Comment:
@cjmash will we need this for the prod env as well?

Reply:
yeah @egichuri , I have updated the prod file

Reply:
great!